packages/fdc3-agent-proxy/src/channels/DefaultChannel.ts

```diff
@@ -24,47 +23,57 @@ export class DefaultChannel implements Channel {
   }
 
   async broadcast(context: Context): Promise<void> {
-    const request: BroadcastRequest = {
-      meta: this.messaging.createMeta(),
-      payload: {
-        channelId: this.id,
-        context,
-      },
-      type: 'broadcastRequest',
-    };
-    await this.messaging.exchange<BroadcastResponse>(request, 'broadcastResponse');
+    const done = await this.messaging.exchange<BroadcastResponse>(
+      {
+        meta: this.messaging.createMeta(),
+        payload: {
+          channelId: this.id,
+          context,
+        },
+        type: 'broadcastRequest',
+      } as BroadcastRequest,
+      'broadcastResponse'
+    );
+    console.log('broadcast done', done);
   }
 
   async getCurrentContext(contextType?: string | undefined): Promise<Context | null> {
-    const request: GetCurrentContextRequest = {
-      meta: this.messaging.createMeta(),
-      payload: {
-        channelId: this.id,
-        contextType: contextType ?? null,
-      },
-      type: 'getCurrentContextRequest',
-    };
-
     // first, ensure channel state is up-to-date
-    const response = await this.messaging.exchange<GetCurrentContextResponse>(request, 'getCurrentContextResponse');
+    const response = await this.messaging.exchange<GetCurrentContextResponse>(
```

Comment:
Why convert `request: GetCurrentContextRequest = { ... }` to `{ ... } as GetCurrentContextRequest`? The former was a safer use of typescript.